src/mds/MDSRank.cc

```diff
@@ -576,9 +576,9 @@ bool MDSRank::handle_deferrable_message(Message *m)
       // SERVER
     case CEPH_MSG_CLIENT_SESSION:
     case CEPH_MSG_CLIENT_RECONNECT:
-      ALLOW_MESSAGES_FROM(CEPH_ENTITY_TYPE_CLIENT);
       // fall-thru
     case CEPH_MSG_CLIENT_REQUEST:
+      ALLOW_MESSAGES_FROM(CEPH_ENTITY_TYPE_CLIENT);
```

Comment:
I understand message types should be derived from CEPH_ENTITY_TYPE_CLIENT rather than the other,
Because the server - > dispatch (m) can only deal with the client type of message
:-)